content/en/integrations/kubernetes_state_core.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Kubernetes State Metrics Core
+name: kubernetes_state_core
+kind: integration
+description: "Monitor the health of the workload running on your Kubernetes cluster. Track the status of your Kubernetes objects, correlate your microservice metrics, and more."
+short_description: "Track the status of your Kubernetes objects, correlate your microservice metrics, and more."
+categories:
+    - cloud
+    - configuration & deployment
+    - containers
+    - orchestration
+doc_link: /integrations/kubernetes_state_core/
+dependencies:
+    ['https://github.com/DataDog/documentation/blob/master/content/en/integrations/kubernetes_state_core.md']
+has_logo: true
+integration_title: Kubernetes State Metrics Core
+is_public: true
+public_title: Datadog-Kubernetes State Metrics Core Integration
+integration_id: "kubernetes_state_core"
+---
+
+## Overview
+
+Get metrics from Kubernetes service in real time to:
+
+- Visualize and monitor kubernetes states
+- Be notified about kubernetes failovers and events.
+
```

Comment:
I think we should add a sentence here to explain how this is different than the Python check

Reply:
what major differences we want to highlight in the overview?  supporting ksm v2 for example?

Reply:
I added more details below about ksm v2 and the performance and tagging improvments

Reply:
Great, maybe also add that this KSM Core check provides a better alternative to the legacy `kubernetes_state`? I think it's not super clear yet